dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -82,6 +82,16 @@ def replace_hostname(url)
   @browser.execute_script("$(\"#{jquery_selector}\").click();")
 end
 
+When /^I press delete$/ do
+  script = "Blockly.mainBlockSpaceEditor.onKeyDown_("
+  script +="{"
+  script +="  target: {},"
+  script +="  preventDefault: function() {},"
+  script +="  keyCode: $.simulate.keyCode['DELETE']"
+  script +="})"
+  @browser.execute_script(script)
+end
```

Comment:
Wanted to use $.simulate (as below in `When /^I hold key`...) but for some reason the browser wasn't picking it up.  So I went straight to the method I needed to test. Suggestions welcome.